ppr-ui/src/components/common/CautionBox.vue

```diff
@@ -0,0 +1,41 @@
+<template>
+  <div :class="$style['caution-box']" class="caution-box">
+    <p class="ma-0 pa-4">
+      <b>Important:</b> {{ msg }}
+    </p>
+  </div>
+</template>
+
+<script lang="ts">
+// external
+import {
+  defineComponent,
+  reactive,
+  toRefs
+} from '@vue/composition-api'
+
+export default defineComponent({
+  name: 'CautionBox',
+  props: {
+    setMsg: String
+  },
+  setup (props) {
+    const localState = reactive({
```

Comment:
quick question, could the message change, ie. is the component reused during the course of the app? If so you will need a watch on this...

Reply:
ha yes I thought about this because of our earlier convo. It will always be static, but it may get initialized with a different message by the parent component

Reply:
10-4